scripts/create_pelorus_operator

```diff
@@ -100,23 +99,19 @@ pushd "${destination_dir}" || exit 2
     echo "INFO: Creating api"
     operator-sdk create api --helm-chart="${source_dir}" || exit 2
 
-# TBD: The symlink is not working. Manager do not contain any charts if they are symlinked
-    # We do want to have symlink to original helm-charts instead of direct copy, 
-    # but ONLY when the pelorus-operator is within pelorus git repository, otherwise leave the copied version
-    # The check assumes the git pelorus charts are one level up from the pelorus-operator folder
-#    if git remote -v | grep \/pelorus.git; then
-#        [ -d "${DEFAULT_OPERATOR_HELM_CHART_SUBDIR}" ] && [ -d "../charts/pelorus" ] && echo "Removing ${DEFAULT_OPERATOR_HELM_CHART_SUBDIR} directory" && rm -rf "${DEFAULT_OPERATOR_HELM_CHART_SUBDIR}"
-#        pushd helm-charts || exit 2
-#            echo "Creating symlink from helm-charts/pelorus to  directory"
-#            ln -s ../"${DEFAULT_PELORUS_CHARTS_SUBDIR}" ./pelorus
-#        popd || exit 2
-#    fi
-
-    # Correct operator version, to be latest +1
-    OPERATOR_VERSION=$(curl -H "Authorization: Bearer XYZ" -X GET "https://quay.io/api/v1/repository/${OPERATOR_ORG_NAME}/${OPERATOR_PROJECT_NAME}/tag/" | jq .tags[].name | head -1 | sed -e 's|\"||g')
-    echo "INFO: Current operator version: ${OPERATOR_VERSION}"
-    NEW_OPERATOR_VERSION=$(echo ${OPERATOR_VERSION} | awk -F. -v OFS=. '{$NF += 1 ; print}')
-    echo "INFO: New operator version: ${NEW_OPERATOR_VERSION}"
+    if [ -z ${operator_ver+x} ]; then
+    destination_dir="${DEFAULT_PELORUS_OPERATOR_DIR}"
+        # Correct operator version, to be latest +1
+        OPERATOR_VERSION=$(curl -H "Authorization: Bearer XYZ" -X GET "https://quay.io/api/v1/repository/${OPERATOR_ORG_NAME}/${OPERATOR_PROJECT_NAME}/tag/" | jq .tags[].name | head -1 | sed -e 's|\"||g')
+        echo "INFO: Current operator version: ${OPERATOR_VERSION}"
+        NEW_OPERATOR_VERSION=$(echo "${OPERATOR_VERSION}" | awk -F. -v OFS=. '{$NF += 1 ; print}')
+        if [ "$NEW_OPERATOR_VERSION" -eq "1" ]; then
+          NEW_OPERATOR_VERSION=0.0.1
+        fi
+    else
```

Comment:
if code enters the else block, does it break line 166? because `OPERATOR_VERSION` was not defined

Reply:
You are correct and the logic is slightly incorrect. Will fix that.

Reply:
Plus I see line 103 is left over from copy-paste...